tests/chainer_tests/test_variable.py

```diff
@@ -507,6 +507,17 @@ def test_unchain(self):
         assert ret[1].rank == old_rank
         self.check_backward((ret[1],), (ret[2],), (ret[3],), False)
 
+    def test_unchain_split(self):
+        if self.x.ndim == 0:
+            return
+        ret = get_variable(np, self.x)
+        ret.grad = np.zeros_like(ret.data)
+        y1, y2 = F.split_axis(ret, [5], axis=0)
+        y1.unchain()
+        z1, z2 = F.sum(y1), F.sum(y2)
+        w = z1 + z2
```

Comment:
Sorry.  I've just found the backward tests in the class don't rely on default grads.  Could you set grad for each variable, including `w`?

Reply:
Okay, I guess I missed it too. Added it.